scripts/bootstrap.sh

```diff
@@ -27,6 +27,7 @@ ulimit -c unlimited
 source "${DIR}/apollo_base.sh"
 
 function start() {
+    nohup cyber_launch start modules/drivers/tools/image_decompress/launch/image_decompress.launch & 
```

Comment:
IIRC, this tool is pretty heavy for online use.
We can leave these kind of steps in the instruction document as a start.

Reply:
Since our simulator send out compressed images, we need this decompress module. Otherwise, traffic light perception does not work correctly.

Reply:
Yeah I understand. How about adding a `scripts/lgsvl_bootstrap.sh` instead of changing the one we use for road test?

Reply:
Added `scripts/bootstrap_lgsvl.sh`.